fdbrpc/FlowTransport.actor.cpp

```diff
@@ -727,8 +727,17 @@ TransportData::~TransportData() {
 	}
 }
 
-ACTOR static void deliver(TransportData* self, Endpoint destination, ArenaReader reader, bool inReadSocket) {
+ACTOR static void deliver(TransportData* self,
+                          Optional<NetworkAddress> peerAddress,
+                          Endpoint destination,
+                          ArenaReader reader,
+                          bool inReadSocket) {
 	TaskPriority priority = self->endpoints.getPriority(destination.token);
+	if (TaskPriority::UnknownEndpoint == priority) {
+		TraceEvent("GotUnknownEndpoint")
```

Comment:
Can we print the flatbuffers file identifier as well maybe?

Reply:
done